src/app/venter/complaints-home/complaints-home.component.css

```diff
@@ -0,0 +1,100 @@
+.container {
+    margin-right: 18%;
+    margin-left: 18%;
+}
+
+.toolbar {
+        padding-top: 100px;
+        padding-bottom: 100px;
+        background-color: blue;
+        align-content: center;
```

Comment:
Are we using flexbox here? I don't think `align-content` does anything otherwise